app/assets/stylesheets/cart.css.scss

```diff
@@ -30,11 +30,12 @@
 }
 
 #cart-table {
-    margin-left:50px;
-    width: 850px;
-    border: none;
-    border-radius: 5px;
-    box-shadow: 0px 10px 20px rgba(0,0,0,.3);
+  margin-left:50px;
+  width: 850px;
+  border: solid 1px #696969;
+  border-radius: 5px 5px 5px 5px;
```

Comment:
Shorthand form for property `border-radius` should be written more concisely as `5px` instead of `5px 5px 5px 5px`